src/UCTNodeRoot.cpp

```diff
@@ -141,16 +141,25 @@ UCTNode* UCTNode::get_nopass_child(FastState& state) const {
     return nullptr;
 }
 
-// Used to find new root in UCTSearch
-UCTNode::node_ptr_t UCTNode::find_child(const int move) {
+// Used to find new root in UCTSearch.
+// BEWARE : caller is now responsible of freeing the return value
```

Comment:
The BEWARE is pretty weird if it returns a unique_ptr!

Reply:
Stale comment - this comment was there when this was a raw pointer return.  Will remove.